forge-game/src/main/java/forge/game/trigger/Trigger.java

```diff
@@ -129,6 +129,10 @@ public String toString(boolean active) {
                 desc = TextUtil.fastReplace(desc,"CARDNAME", CardTranslation.getTranslatedName(currentName));
                 desc = TextUtil.fastReplace(desc,"NICKNAME", Lang.getInstance().getNickName(CardTranslation.getTranslatedName(currentName)));
             }
+            if (desc.contains("TRIGGERSOURCE")) {
```

Comment:
should use ORIGINALHOST like all the other ones
looks like _Nettlevine Blight_ was already expecting it to exist 🐣

Reply:
maybe it can be moved from `replaceAbilityText` because that should always use this result anyway?